### ISSUE TITLE
Allow searching for service accounts on actors page

### ISSUE THREAD

**Issue by jamilbk** [CloudFire-LLC/cloudfire#6146]
> Feedback: its hard to search for service-accounts in the actors page, theres not filter for that

**Comment by jamilbk** [comment]
I'm a bit perplexed by this one: AFAIK we do support this already. Might just need to verify.

**Comment by bmanifold** [comment]
@jamilbk, yes our current search does include the service accounts, but I'm wondering if this is similar to what I had asked about the Policy search functionality....I'm wondering if they want to be able to specify "when I search only search for this type of actor" e.g. admin/user/service_account/etc...

**Comment by bmanifold** [comment]
I wonder if a better thing to add would be a button group (or potentially a `select`), similar to the `All / Enabled / Disabled' buttons that filter things down to a certain type.

**Comment by jamilbk** [comment]
> I wonder if a better thing to add would be a button group (or potentially a select), similar to the `All / Enabled / Disabled' buttons that filter things down to a certain type.

Yeah so a generalized rule would be then to first filter by the buttons, then apply filer search text to the remaining results?

**Comment by bmanifold** [comment]
> Yeah so a generalized rule would be then to first filter by the buttons, then apply filer search text to the remaining results?

Yeah, that would be my basic thinking.  Default the buttons to `all` and you can search everything at once if you want, but if you want a more targeted search select a button/dropdown first, then search.
